packages/scene-composer/src/store/slices/SceneDocumentSlice.ts

```diff
@@ -348,6 +355,14 @@ export const createSceneDocumentSlice = (set: SetState<RootState>, get: GetState
         }
         draft.lastOperation = 'updateComponentInternal';
       });
+
+      const updatedComponenet = get().document?.nodeMap[nodeRef]?.components[componentToUpdateIndex];
+      if (
+        (componentType === 'Tag' || componentType === 'DataOverlay' || componentType === 'EntityBinding') &&
```

Comment:
use `KnownComponentType.Tag` instead of `'Tag'` to avoid typo. same for other types